cmake/modules/add_python_tests.cmake

```
@@ -37,15 +37,26 @@ function(add_python_tests)
     find_package(Python3 REQUIRED COMPONENTS Interpreter)
   endif()
 
-  # Get the list of tests from pytest - force rootdir to be the working directory
+  # Set rootdir to the working directory to limit the search, and for relative test names
+  # Set confcutdir to the working directory to ignore top level conftest.py
+  set(PYTEST_IGNORE_ROOT_STR "--rootdir=${PYTEST_WORKING_DIRECTORY} --confcutdir=${PYTEST_WORKING_DIRECTORY}")
```

Nitpick, `IGNORE` to me implies skipping tests in a given set whereas this is specifying the test set in the first place, suggest just `PYTEST_ROOT_STR` or similar (here and below)

```suggestion
  # Set rootdir to the working directory to limit the search, and for relative test names
  # Set confcutdir to the working directory to ignore top level conftest.py
  set(PYTEST_ROOT_STR "--rootdir=${PYTEST_WORKING_DIRECTORY} --confcutdir=${PYTEST_WORKING_DIRECTORY}")
```